common/script/cron.js

```diff
@@ -41,7 +41,7 @@ function sanitizeOptions (o) {
     timezoneOffset = timezoneOffsetDefault;
   }
 
-  let now = o.now ? moment(o.now).zone(timezoneOffset) : moment().zone(timezoneOffset);
+  let now = o.now ? moment(o.now).utcOffset(timezoneOffset) : moment().utcOffset(timezoneOffset);
```

Comment:
@SabreCat the problem is that our data could not be compatible since utcOffset doesn't always return the same data as zone()

I think @Alys wanted to change this but wasn't able to